### PR TITLE
Updated the anchors in the index file

### PR DESCRIPTION
**Pull request by dbowiecag**
Had to update the anchors so the url will be >index.html#request

Thanks!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,49 +1,53 @@
...
-    <title>Media Department | Cornerstone Church </title>
-    <meta name='viewport' content='initial-scale=1.0'>
-    <meta name="apple-mobile-web-app-capable" content="yes">
+	<title>Media Department | Cornerstone Church </title>
+	<meta name='viewport' content='initial-scale=1.0'>
+	<meta name="apple-mobile-web-app-capable" content="yes">
 	<link rel="apple-touch-icon" href="touch-icon-iphone.png">
 	<link rel="apple-touch-icon" sizes="152x152" href="touch-icon-ipad.png">
 	<link rel="apple-touch-icon" sizes="180x180" href="touch-icon-iphone-retina.png">
 	<link rel="apple-touch-icon" sizes="167x167" href="touch-icon-ipad-retina.png">
```
Comment (silentsonata) [inline index.html:11]: LINES: 8-11

What are these for? Are they needed?

**Review by silentsonata — COMMENTED** [review]
There is a section of code that I am not sure what it is for. Do we need it?

**Comment by dbowiecag** [comment]
That was transferred from my landing template. It’s used for apple devices pulling proper fav icons on different resolutions

File: index.html (hunk 1 above)
Comment (dbowiecag) [reply index.html:11]: Transferred from the template I made on other sites. Used for fav icons on different devices.

Comment (silentsonata) [reply index.html:11]: Alright can we remove them?

**Review by silentsonata — CHANGES_REQUESTED** [review]
Remove the unnecessary <link>

File: index.html (hunk 1 above)
Comment (silentsonata) [reply index.html:11]: Do we have a status on this pull request?

**Comment by silentsonata** [comment]
Code looks fine.

**Pull request merged**